app/Main.hs

```diff
@@ -65,5 +70,47 @@ numberWang = do
     putStrLn "Aah, would that it were NumberWang. Alas, it is not."
```

Comment:
I think we should also show the correct value here in the error message. What do you think? Would it help learners?

Reply:
I actually had the correct translation displayed before but decided to take it out since `genNumbers` wasn't implemented yet. I think I can add an option to the CLI, once I start working on it, asking the user whether they want the see answers or not and leave this as is for now.

Reply:
Asking user for this sounds good!

Reply:
You can open an issue to make this a feature request, if you wish.